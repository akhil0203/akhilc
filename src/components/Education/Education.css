.Education {
  width: 100%;
  height: '600px';
  background-image: url(../../images/backgrounds/education.jpg);
}
