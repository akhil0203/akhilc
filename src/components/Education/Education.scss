.education {
  background-image: url(../../images/backgrounds/education.jpg);
  background-size: cover;
  display: flex;

  > .col {
    padding: 150px 0 150px 0;
  }

  .col > .row:not(:last-child) {
    margin-bottom: 40px;
  }

  .row {
    justify-content: center;
    align-items: center;

    h2 {
      color: #fff;
      font-size: 40px;
      font-weight: 700;
      text-transform: uppercase;
    }

    h3 {
      color: #fff;
      font-size: 30px;
      font-weight: 700;
      text-align: center;
    }

    p {
      color: #fff;
      font-size: 16px;
      font-weight: 700;
      text-align: center;
    }
  }
}
