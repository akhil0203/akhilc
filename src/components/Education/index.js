import React from 'react';
import { Container, Col, Row } from 'reactstrap';
import { Event } from '../Shared/Tracking';

import senecaImg from '../../images/thumbnails/companies/seneca.png';

import './styles.scss';

const Education = () => {
  return (
    <Container className="education" fluid>
      <Col>
        <Row>
          <h2>Education</h2>
        </Row>
        <Col>
          <Row>
            <Col sm={4}>
              <Row>
                <a
                  href="https://bit.ly/seneca_bsd"
                  target="seneca-bsd"
                  onClick={() =>
                    Event(
                      'Education',
                      'Clicked Seneca BSD link',
                      'Seneca BSD link',
                    )
                  }
                >
                  <img src={senecaImg} alt="seneca" />
                </a>
              </Row>
              <Row>
                <h3>Software Development</h3>
              </Row>
              <Row>
                <p>Seneca College, Toronto ON</p>
              </Row>
            </Col>
            <Col sm={4}>
              <Row>
                <a
                  href="https://bit.ly/seneca_ect"
                  target="seneca-ect"
                  onClick={() =>
                    Event(
                      'Education',
                      'Clicked Seneca ECT link',
                      'Seneca ECT link',
                    )
                  }
                >
                  <img src={senecaImg} alt="seneca" />
                </a>
              </Row>
              <Row>
                <h3>Computer Engineering</h3>
              </Row>
              <Row>
                <p>Seneca College, Toronto ON</p>
              </Row>
            </Col>
          </Row>
        </Col>
      </Col>
    </Container>
  );
};

export default Education;
