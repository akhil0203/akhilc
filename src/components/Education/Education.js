import React from 'react';

import './Education.css';

const Education = () => {
  return (
    <div className="Education">
      <h1>EDUCATION</h1>
    </div>
  );
};

export default Education;
