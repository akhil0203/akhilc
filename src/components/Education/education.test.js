import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Education from '.';

it('matches the snapshot', () => {
  const tree = renderer.create(<Education />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders the Education component without crashing', () => {
  shallow(<Education />);
});

it('renders the main header', () => {
  const wrapper = shallow(<Education />);

  expect(wrapper.find('h2').html()).toEqual('<h2>Education</h2>');
});

it('renders details for the BSD program', () => {
  const wrapper = shallow(<Education />);

  expect(
    wrapper
      .find('Col>Row>Col')
      .at(0)
      .html(),
  ).toEqual(
    '<div class="col-sm-4"><div class="row"><a href="https://bit.ly/seneca_bsd" target="seneca-bsd"><img src="seneca.png" alt="seneca"/></a></div><div class="row"><h3>Software Development</h3></div><div class="row"><p>Seneca College, Toronto ON</p></div></div>',
  );
});

it('renders details for the ECT program', () => {
  const wrapper = shallow(<Education />);

  expect(
    wrapper
      .find('Col>Row>Col')
      .at(1)
      .html(),
  ).toEqual(
    '<div class="col-sm-4"><div class="row"><a href="https://bit.ly/seneca_ect" target="seneca-ect"><img src="seneca.png" alt="seneca"/></a></div><div class="row"><h3>Computer Engineering</h3></div><div class="row"><p>Seneca College, Toronto ON</p></div></div>',
  );
});
