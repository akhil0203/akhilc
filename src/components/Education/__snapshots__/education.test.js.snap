// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="education container-fluid"
>
  <div
    className="col"
  >
    <div
      className="row"
    >
      <h2>
        Education
      </h2>
    </div>
    <div
      className="col"
    >
      <div
        className="row"
      >
        <div
          className="col-sm-4"
        >
          <div
            className="row"
          >
            <a
              href="https://bit.ly/seneca_bsd"
              onClick={[Function]}
              target="seneca-bsd"
            >
              <img
                alt="seneca"
                src="seneca.png"
              />
            </a>
          </div>
          <div
            className="row"
          >
            <h3>
              Software Development
            </h3>
          </div>
          <div
            className="row"
          >
            <p>
              Seneca College, Toronto ON
            </p>
          </div>
        </div>
        <div
          className="col-sm-4"
        >
          <div
            className="row"
          >
            <a
              href="https://bit.ly/seneca_ect"
              onClick={[Function]}
              target="seneca-ect"
            >
              <img
                alt="seneca"
                src="seneca.png"
              />
            </a>
          </div>
          <div
            className="row"
          >
            <h3>
              Computer Engineering
            </h3>
          </div>
          <div
            className="row"
          >
            <p>
              Seneca College, Toronto ON
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
